components/cell/jwks-server/crypto/service/unsecured.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package service
+
+import (
+	"encoding/json"
+	"github/gorilla/mux"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+const httpPort int = 8186
```

Comment:
We have tests for default ports. Do we have tests when ports are set through environment variables ?

Reply:
Yes. 
Fixed by #75